x-pack/plugins/apm/public/components/app/Main/route_config/index.tsx

```diff
@@ -294,15 +293,8 @@ export const routes: APMRouteDefinition[] = [
     exact: true,
     path: '/services/:serviceName/nodes/:serviceNodeName/metrics',
     component: withApmServiceContext(ServiceNodeMetrics),
-    breadcrumb: ({ match }) => {
-      const { serviceNodeName } = match.params;
-
-      if (serviceNodeName === SERVICE_NODE_NAME_MISSING) {
-        return UNIDENTIFIED_SERVICE_NODES_LABEL;
-      }
-
-      return serviceNodeName || '';
-    },
+    breadcrumb: ({ match }) =>
+      getServiceNodeName(match.params.serviceNodeName) || '',
```

Comment:
`getServiceNodeName` will never return `undefined` afaict

```suggestion
      getServiceNodeName(match.params.serviceNodeName),
```